backend/src/entity/user.ts

```diff
@@ -0,0 +1,21 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+/* eslint-disable indent*/
+
+@Entity('user')
+export class User {
+    @PrimaryGeneratedColumn({ type: 'bigint' })
+    id: number;
+
+    @Column({ type: 'varchar', length: 255, nullable: false })
+    email: string;
+
+    @Column({ type: 'varchar', length: 255, nullable: false })
+    token: string;
+
+    @Column({ type: 'bigint', array: true, default: '{}' })
+    projects_id: number[];
+
+    @Column({ type: 'boolean', nullable: false })
+    role: boolean;
```

Comment:
Why is a 'role' in boolean ? Either rename this to 'superUser' or 'admin' or change its type with a relation table to a role entity